Common/Securities/Future/FuturesExpiryUtilityFunctions.cs

```diff
@@ -36,21 +35,20 @@ public static class FuturesExpiryUtilityFunctions
         /// <returns>The date-time after adding n business days</returns>
         public static DateTime AddBusinessDays(DateTime time, int n, bool useEquityHolidays = true, IEnumerable<DateTime> holidayList = null)
         {
-
-            var holidays = holidayList?.ToList() ?? new List<DateTime>();
+            var holidays = holidayList?.Select(x => x.Date).ToList() ?? new List<DateTime>();
```

Comment:
A minor concern, but here we expect `holidayList` to potentially have a `time` component but the same isn't expected in the other methods `NthLastBusinessDay` & `NthBusinessDay` having the same argument, think we should standardize it to this behavior?

Reply:
@Martin-Molinero - I had wondered about that as I was making the change. Happy to update.

Reply:
Agree 👍 and avoid any deja vu of this bug

Reply:
@Martin-Molinero - updated with changes to other methods and tests